harc_game_web/apps/tasks/models.py

```diff
@@ -39,16 +39,16 @@ class DocumentedTask(models.Model):
     """
     Model udokumentowanego wykonanego zadania
     """
-    task = models.ForeignKey(Task, on_delete=models.RESTRICT, null=True, default=None, related_name='documented_tasks')
```

Comment:
A dlaczego to wypada?

Reply:
sprawiało problemy a nie jest konieczne 🤷

Reply:
(po make clean i zrobieniu migracji wywalały błędy, usunięcie tego i analogiczne zmiany poprawiły sytuację, a myślę że nie wpływają znacząco na jakość kodu)